WcaOnRails/config/routes.rb

```diff
@@ -346,6 +345,12 @@
         get '/group-type/:group_type' => 'roles#index_for_group_type', as: :index_for_group_type
       end
       resources :user_groups, only: [:index, :create, :update]
+      namespace :wrt do
+        resources :persons, only: [:update, :destroy]
+        scope 'persons' do
```

Comment:
You can directly enter the namespace by introducing a `do` block in the `resources` call.